package.json

```diff
@@ -1,11 +1,9 @@
 {
   "name": "modtree",
   "version": "0.1.0",
+  "license": "MIT",
+  "scripts": {},
```

Comment:
duplicate `scripts` key here

(not sure why multiline comment isn't working for this comment)